src/CSharp.Mongo.Migration/Core/MigrationRunner.cs

```diff
@@ -105,6 +132,8 @@ public class MigrationRunner : IMigrationRunner {
         MigrationDocument document = migration.ToDocument();
         await _migrationCollection.InsertOneAsync(document);
 
+        _logger.LogInformation("Completed running migration with version: '{version}'", migration.Version);
```

Comment:
@JordanDChappell I think it would be useful to include start/end or execution time, not a blocker though.

Reply:
@yoshifp Maybe we can leave this to the consumer to configure as part of the logger that they register? I believe that different loggers can be configured to include timestamps and other information, here's an example from the Microsoft doco: https://learn.microsoft.com/en-us/dotnet/core/extensions/console-log-formatter